### PR TITLE
SPDX-CTL command documentation and cookbook_upload change

### PR DESCRIPTION
**Pull request by msys-sgarg**
Signed-off-by: smriti <sgarg@msystechnologies.com>

### Description

SPDX license linking ctl commands need to have related documentation as well. 

### Issues Resolved

https://github.com/chef/supermarket/issues/2188

### Check List

- [x] New functionality includes tests
- [x] All buildkite tests pass
- [x] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [x] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by msys-sgarg** [comment]
https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/304 - Build done

File: src/supermarket/app/controllers/api/v1/cookbook_uploads_controller.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def create
           UniverseCache.flush
 
           # fetch spdx license
-          SpdxLicenseUpdateWorker.perform_async(FetchSpdxLicenseJson.spdx_license_json, cookbook_version.version)
+          SpdxLicenseUpdateWorker.perform_async(FetchSpdxLicenseJson.spdx_license_json, cookbook_version.id)
```
Comment (RajeshPaul38) [inline src/supermarket/app/controllers/api/v1/cookbook_uploads_controller.rb:70]: we should try using named arguments to make things more clear.

File: docs-chef-io/content/supermarket/ctl_supermarket.md [hunk 2]
```diff
@@ -29,6 +29,43 @@ sudo -u supermarket supermarket-ctl make-admin USER_NAME
 where `USER_NAME` represents the name of the user to be granted
 administrator privileges.
 
+## SPDX license update
+
```
Comment (tas50) [inline docs-chef-io/content/supermarket/ctl_supermarket.md:33]: We should make sure to document the purpose of the SPDX commands. Why does Supermarket include SPDX data and why would a person ever need to run these commands?

Comment (msys-sgarg) [reply docs-chef-io/content/supermarket/ctl_supermarket.md:33]: That we have already covered as part of this PR - https://github.com/chef/supermarket/pull/2296/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 here -- actually we missed SPDX command documentation in earlier documentation and hence raised this PR separately

Comment (tas50) [reply docs-chef-io/content/supermarket/ctl_supermarket.md:33]: The readme isn't for users though. The readme is for developers. The docs site is where all user facing documentation should live.

Comment (msys-sgarg) [reply docs-chef-io/content/supermarket/ctl_supermarket.md:33]: ohh ok .. I will make the required changes in documentation here

**Review by tas50 — APPROVED** [review]

**Pull request merged**